www/include/monitoring/status/Common/commonJS.php

```diff
@@ -41,8 +41,20 @@
     include_once "./include/monitoring/status/Common/default_poller.php";
 }
 
+// Get the current header version to define the correct header id in javascript
+$headerVersion = 1;
+$userFeatures = $centreonFeature->userFeaturesValue(
+    $centreon->user->get_id()
+);
+foreach($userFeatures as $feature) {
+    if($feature['name'] == 'Header' && $feature['enabled'] == 1) {
```

Comment:
please add a space after the "foreach" and the "if" and split the line after the "&&" -> 
'foreach ($userFeatures as $feature) {
''''    if ($feature['name'] == 'Header' &&
'''' ''''        $feature['enabled'] == 1
''''    ) {
'''' '''        $headerVersion = (int) $feature['version'];
''''    }
'}